README.md

```diff
@@ -19,3 +19,19 @@ steps:
       appSecret: ${{ secrets.APP_SECRET }}
       apkPath: 'path/to/apk'
 ```
+
+### All Params
```

Comment:
Let's also add a link to the Oculus documentation for each directly in the table and a link to the top of the documents under this title. 
```suggestion
### All Parameters
```